src/bound/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod bound;
+
+pub use self::bound::*;
```

Comment:
so, I think really just tree and db should be split up for now. also feel free to rename PageMaterializer to just Materializer because it's namespaced to page.rs already.